invenio/ext/sqlalchemy/testsuite/test_composite_dicts.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Unit tests for add_orcid."""
+
+from nose_parameterized import parameterized, param
+from sqlalchemy import Column, String, Integer
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.ext.mutable import MutableComposite
+from sqlalchemy.orm import composite
+from unittest import TestCase
+
+from invenio.ext.sqlalchemy.utils import get_model_type
+from invenio.testsuite import make_test_suite
+
+
+Base = declarative_base()
+Base = get_model_type(Base)  # Patch with our methods
+
+
+class MyComposite(MutableComposite):
+
```

Comment:
Thanks a lot ! It can seem picky but, I didn't realize when I was reading the code I was in the test area. (I admit I skipped to check the import order, (BTW I think that from unittest import TestCase should be on top)), so for my next time I will directly see that it is a test classes ;) ! Thanks again